Tools/electrostatic_sphere_plot.py

```diff
@@ -0,0 +1,149 @@
+import yt
+import matplotlib.pyplot as plt
+from scipy.optimize import fsolve
+from skimage import measure
+import numpy as np
+
+yt.funcs.mylog.setLevel(50)    #Mute yt's verbose shenanigans
```

Comment:
Usually like this in python: two whitespaces before inline comment, one space after `#`
```suggestion
yt.funcs.mylog.setLevel(50)  # mute yt's verbose shenanigans
```